__tests__/integration/select.test.ts

```diff
@@ -328,35 +337,14 @@ describe('select', () => {
         format: 'JSON',
       })
       try {
-        expect(() => result.stream()).toThrowError(
+        await expect(async () => result.stream().next()).rejects.toThrowError(
           'JSON format is not streamable'
         )
       } finally {
         result.close()
       }
     })
 
-    it('can pause response stream', async () => {
-      const result = await client.query({
-        query: 'SELECT number FROM system.numbers LIMIT 10000',
-        format: 'CSV',
-      })
-
-      const stream = result.stream()
-
-      let last = null
-      let i = 0
-      for await (const chunk of stream) {
-        last = chunk.text()
-        i++
-        if (i % 1000 === 0) {
-          stream.pause()
-          setTimeout(() => stream.resume(), 100)
-        }
-      }
-      expect(last).toBe('9999')
-    })
-
```

Comment:
Probably not required anymore with async iterators. It can be paused (not consumed) on the application level, as it is a lazy evaluation.